rest/common/utils/rest.py

```diff
@@ -0,0 +1,339 @@
+"""
+Python module for initiating and executing commands via REST API.
+"""
+
+import json
+import time
+
+import requests
+from dotenv import dotenv_values
+
+from rest.common.config.config import Config
+from rest.common.utils.exceptions import CommandExecutionError, HTTPError
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def rest():
+    """
+    REST method to be used by any test method
+    """
+    config = dotenv_values(
+        ".env"
```

Comment:
probably specific path would be good.

Reply:
reading from .env is the standard practice